packages/cube/test/cube/constraintBuilder/support.js

```diff
@@ -0,0 +1,38 @@
+import { strictEqual } from 'assert'
+import rdf from 'barnard59-env'
+import SHACLValidator from 'rdf-validate-shacl'
+
+const { sh } = rdf.ns
+
+const subject = rdf.namedNode('http://example.org/subject')
+const property = rdf.namedNode('http://example.org/property')
+
+const createDataset = values =>
+  rdf.clownface().namedNode(subject).addOut(property, values).dataset
+
+export const conforms = (validator, values) => {
+  const report = validator.validate(createDataset(values))
+  strictEqual(report.conforms, true)
+}
+
+export const notConforms = (validator, values) => {
+  const report = validator.validate(createDataset(values))
+  strictEqual(report.conforms, false)
+}
```

Comment:
Have you considered variadic arg for `values`? This would avoids arrays when calling in tests

```suggestion
export const conforms = (validator, ...values) => {
  const report = validator.validate(createDataset(values))
  strictEqual(report.conforms, true)
}

export const notConforms = (validator, ...values) => {
  const report = validator.validate(createDataset(values))
  strictEqual(report.conforms, false)
}
```

Reply:
done